processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -149,6 +150,17 @@ public boolean supportsBitmapIndex(BitmapIndexSelector selector)
   }
 
   @Override
+  public double estimateSelectivity(ColumnSelector columnSelector, BitmapIndexSelector indexSelector)
+  {
+    // Estimate selectivity with attribute value independence assumption
```

Comment:
Let's ensure there won't be regression if users don't touch any configurations, just update druid binary to the version which includes this patch. "No regression if user has to manually change some config" is a regression, IMO